### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by asharnadeem**

### CODE REVIEW

File: students/an3056.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
+| UNI            | an3056                                                                 |
+| GitHub Profile | [link](https://github.com/asharnadeem)                                 |
+| GitHub README  | [link](https://github.com/asharnadeem/asharnadeem/blob/main/README.md) |
 
 ## Project Proposal
 
-- Create a component library that offers users prebuilt React components to build sleek, innovative UI's with inspiration from innovators in the space like [Linear](https://linear.app/) and [Stripe](https://stripe.com/).
-
-- Build a type safe, all in one utility library for Typescript that makes working with primitive data types very easy similar to what exists in the Java ecosystem with the [Apache Common Utils](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/).
+- [tool-kit](https://github.com/asharnadeem/tool-kit)
```
Comment (timkpaine) [inline students/an3056.md:12]: relative path to markdown file

File: students/an3056.md [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 
 ## Project Contributions
 
-- [Just](https://github.com/angus-c/just)
-- [Prisma](https://github.com/prisma/prisma)
-- [Typescript](https://github.com/Microsoft/TypeScript)
+- [faker](./projects/python/faker.md)
```
Comment (timkpaine) [inline students/an3056.md:16]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/an3056.md (hunk 2 above)
Comment (asharnadeem) [reply students/an3056.md]: Addressed

File: students/an3056.md (hunk 1 above)
Comment (asharnadeem) [reply students/an3056.md]: Addressed

**Comment by timkpaine** [comment]
lgtm

**Review by timkpaine — APPROVED** [review]

**Pull request merged**